app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,149 @@
  *= require_tree .
  *= require_self
  */
+body {
+  background-color: burlywood;
+}
+
+.flex-col {
+  width: 20%;
+  height: auto;
+  border: solid 2px;
+  margin-left: 5%;
+  margin-top: -1.5%;
+}
+
+.user-photo {
+  margin-left: 1%;
+}
+
+.container-header {
+  margin-bottom: 1%;
+}
+
+.user-totalpost {
+  margin-left: 30%;
+}
+
+.header-title {
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.user-card {
+  width: 30%;
+  height: auto;
+  border: solid 2px;
+  margin-left: 1%;
+  padding-bottom: 1%;
+}
+
+.bio-title {
+  margin-bottom: 5%;
+  margin-left: 1%;
+  display: block;
+  font-weight: bold;
+}
+
+.user-bio {
+  margin-top: 5%;
+  margin-left: 1%;
+}
+
+.post-container {
+  display: flex;
+  width: 30%;
+  height: auto;
+  border: solid 2px;
+  margin-left: 1%;
+  margin-top: 1%;
+}
+
+.post-title {
+  margin-bottom: 5%;
+  margin-left: 1%;
+  font-weight: bold;
+}
+
+.post-bio {
+  margin-top: 8%;
+  margin-left: -8.5%;
+  white-space: nowrap;
+}
+
+.post-totalpost {
+  margin-top: 12%;
+  margin-left: 25%;
+}
+
+.see-all-posts {
+  width: 10%;
+  font-size: 16px;
+  margin-top: 2%;
+  background-color: burlywood;
+  margin-left: 10%;
+}
+
+.no-post {
+  margin-left: 5%;
+  color: rgb(225, 22, 22);
+  font-size: 18px;
+}
+
+.no-comment {
+  margin-left: 5%;
+  color: rgb(225, 22, 22);
+  font-size: 18px;
+  margin-top: 1%;
+}
+
+.pagination {
+  width: 10%;
+  font-size: 16px;
+  margin-top: 2%;
+  background-color: burlywood;
+  margin-left: 10%;
+}
+
+.post-details {
+  display: flex;
+  width: 30%;
+  height: auto;
+  border: solid 2px;
+  margin-left: 1%;
+  margin-top: 1%;
+}
+
+.comment-totalpost {
+  margin-top: 12%;
+  margin-left: 25%;
+}
+
+.post-comment-1 {
+  margin-top: 15%;
+  margin-left: -48%;
+}
+
+.comment-title {
+  margin-left: 1%;
+}
+
+.comment-count {
+  margin-top: 25%;
+  margin-left: 120%;
+  white-space: nowrap;
+}
+
+.like-count {
+  margin-top: -38%;
+  margin-left: 240%;
+  white-space: nowrap;
+}
+
+.recent-comments {
+  margin-left: 2%;
+}
+
+.all-comments {
+  margin-left: 2%;
+}
```

Comment:
- [ ] Please kindly avoid code repetition in order for your codebase to be neat. ℹ️